src/pages/cli/migration/aws-cdk-migration.mdx

```diff
@@ -0,0 +1,44 @@
+export const meta = {
+  title: `AWS CDK v1 to v2 migration`,
+  description: ``,
+};
+
+<Callout>
+AWS CDK version should match the CDK version that cli-extensibility-helper package uses as a dependency in `amplify/backend/custom/customResourceName/package.json`.
+
+
+You can verify the `aws-cdk-lib` version that `@aws-amplify/cli-extensibility-helper` uses in the `package.json`  [here](https://www.npmjs.com/package/@aws-amplify/cli-extensibility-helper?activeTab=explore).
+
+</Callout>
+
+AWS Cloud Development Kit [CDK version 1](https://docs.aws.amazon.com/cdk/v2/guide/home.html) that Amplify CLI uses under the hood has entered maintenance on June 1, 2022 with end of support date scheduled on 6/1/2023.
```

Comment:
```suggestion
[AWS Cloud Development Kit (CDK) version 1](https://docs.aws.amazon.com/cdk/v2/guide/home.html) that Amplify CLI uses under the hood has entered maintenance on June 1, 2022 with end of support date scheduled on June 1, 2023.
```

Should the link point to the cdk v1 docs?

Reply:
cc @akshbhu I've committed this change but not sure if the link should be updated